docs/security.md

```diff
@@ -66,6 +66,23 @@ We use a number of static analysis tools to identify security risks in developme
 
 All development laptops have encrypted hard drives.
 
+## Updates
+
+Sourcegraph follows commonly recognized best practices for updating software dependencies.
```

Comment:
```suggestion
Sourcegraph follows commonly-recognized best practices for updating software dependencies.
```

Reply:
Adverbs ending in `ly` should not be connected to the adjective they modify with a hyphen: https://prowritingaid.com/grammar/1000138/How-to-use-hyphen-after-adverbs.